parquet/src/main/java/org/apache/iceberg/parquet/Parquet.java

```diff
@@ -170,6 +170,11 @@ public WriteBuilder overwrite(boolean enabled) {
       return this;
     }
 
+    public WriteBuilder writerVersion(WriterVersion version) {
```

Comment:
Is this mostly for testing? Or is it something we want folks to be using in general? Just wondering if this should be public

Reply:
This provides users a way to create Parquet files with different formats. Currently only used for testing, though. I don't see harm in leaving it public.